peercrawler.py

```diff
@@ -432,29 +433,17 @@ def run_peer_service_forever(peerman, addr='', port=7070):
                 conn.sendall(data)
 
 
-def get_peers_from_service(ctx, addr='::ffff:78.46.80.199'):
-    with socket.socket(socket.AF_INET6, socket.SOCK_STREAM) as s:
-        s.setsockopt(socket.IPPROTO_IPV6, socket.IPV6_V6ONLY, 0)
-        s.settimeout(10)
-        try:
-            s.connect((addr, ctx['peercrawlerport']))
-            response = readall(s)
-            hdr = peer_service_header.parse(response[0:peer_service_header.size])
-            assert (hdr.protocol_ver == 3)
-            if hdr.net_id != ctx['net_id']:
-                raise PeerServiceUnavailable("Peer service for the given network is unavailable")
-        except (PyNanoCoinException, OSError, TypeError) as e:
-            print("Error getting peers: %s" % str(e))
-            raise PeerServiceUnavailable("Peer service is unavailable")
+def get_peers_from_service(ctx: dict, url: str ='http://hetzner1.siganos.xyz:5001/peercrawler/json'):
```

Comment:
I don't think it's a good idea to hardcode this, at least not here.

Reply:
It can be hardcoded in command line argument parser